starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecBridge.java

```diff
@@ -986,16 +986,15 @@ private int queueSecureInputBuffer(
                 + "CryptoException.ERROR_INSUFFICIENT_OUTPUT_PROTECTION");
         return MediaCodecStatus.INSUFFICIENT_OUTPUT_PROTECTION;
       }
-      Log.e(
-          TAG,
-          "Failed to queue secure input buffer, CryptoException with error code "
-              + e.getErrorCode());
+      Log.e(TAG, "Failed to queue secure input buffer. Error code %d", errorCode, e);
```

Comment:
The upstream seems to use a different formatting (which can probably be refined), and the same to line 966.  Try to check if the upstream can be updated with an improved formatting (i.e. if its default code formater will generate a better version).